apps/patient/src/views/Pharmacy.tsx

```diff
@@ -305,6 +300,62 @@ export const Pharmacy = () => {
     }
   };
 
+  const setPreferredPharmacy = async (patientId: string, pharmacyId: string) => {
```

Comment:
wdyt about putting this mutation in an API folder? I read [this](https://www.apollographql.com/blog/apollo-client/architecture/client-side-architecture-basics/#should-we-include-graphql-mutations-in-our-presentation-components) and he suggests queries in component and mutations separate. I kinda like the logic

Reply:
Yeah i like that! was looking for ways to cleanup, getting a bit crazy. Today is also crazy, but gonna try to get this out. I'll followup with some refactoring for this.